payment-manager/src/main/java/com/qpay/paymentmanager/model/entity/wallet/WalletEntity.java

```diff
@@ -0,0 +1,47 @@
+package com.qpay.paymentmanager.model.entity.wallet;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.EnumType;
+import jakarta.persistence.Enumerated;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+import org.qpay.models.UserType;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Entity
+@Builder
+@Table(name = "wallets")
+public class WalletEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    private String name;
+
+    private BigDecimal balance;
+
+    private long userId;
+
+    @Enumerated(EnumType.STRING)
+    private UserType userType;
```

Comment:
how do you use `UserType` if it does not exist?

Reply:
i thought `libs` project should be parallel to qpay and obviously it worked only in my PC, so i moved this `libs` into qpay